src/main/kotlin/com/openlattice/linking/LinkingConfiguration.kt

```diff
@@ -43,6 +44,8 @@ private val DEFAULT_ENTITY_TYPES = setOf("general.person")
  */
 @ReloadableConfiguration(uri = "linking.yaml")
 data class LinkingConfiguration(
+        @JsonProperty("searchConfiguration") val searchConfiguration: SearchConfiguration,
+        @JsonProperty("reportEmailAddress") val errorReportingEmail: String,
```

Comment:
0 importance but would prefer for these to be consts up top like the others for consistency

Reply:
Ah, yes, makes sense